lib/format.ex

```diff
@@ -296,6 +323,28 @@ defmodule Strftime.Format do
     Integer.mod(Calendar.ISO.date_to_iso_days(year, month, day) + 5, 7)
   end
 
+  defp year_day_number(year, month, day) do
+    Calendar.ISO.date_to_iso_days(year, month, day) - Calendar.ISO.date_to_iso_days(year, 1, 0)
+  end
+
+  defp week_number(year, month, day, day_of_week_one_fun) do
+    days_from_week_one = year_day_number(year, month, day) - day_of_week_one_fun.(year)
+
+    case days_from_week_one do
+      days when days < 0 -> 0
+      days -> div(days, 7) + 1
+    end
+  end
+
+  defp day_of_first_monday(year) do
+    first_wday = :calendar.day_of_the_week(year, 1, 1)
```

Comment:
We sometimes use `:calendar.day_of_the_week` and other times `Calendar.ISO.day_of_week`. Do the do the same? If so, should we settle on one of them?

Reply:
I thought I went over all places I used `:calendar`. I think we should prefer `Calendar.ISO` when we can. Will push an update.